pkgs/applications/networking/cluster/terraform/default.nix

```diff
@@ -118,4 +120,23 @@ in rec {
   });
 
   terraform_0_11-full = terraform_0_11.withPlugins lib.attrValues;
+
+  # Tests that the plugins are being used. Terraform looks at the specific
+  # file pattern and if the plugin is not found it will try to download it
+  # from the Internet. With sandboxing enable this test will fail if that is
+  # the case.
```

Comment:
```
export HTTP_PROXY=127.0.0.1:0 HTTPS_RPOXY=127.0.0.1:0
```

might also make it work without relying on the sandbox.

Reply:
sounds good, added